src/extension/fake_id0/open.c

```diff
@@ -94,3 +84,41 @@ int handle_open_enter_end(Tracee *tracee, Reg fd_sysarg, Reg path_sysarg,
 	return 0;
 }
 
+/** Handles open, openat, and creat syscalls. Creates meta files to match the
```

Comment:
This comment seems a little disingenuous, and it comes down again to the naming convention of writing/reading "meta files". It seems like that's not really what we're trying to accomplish anymore.